setup.py

```diff
@@ -215,6 +218,10 @@ def run(self):
                 os.path.join(SRC_DIR, '{}.pb.cc'.format(proto_base)),
                 os.path.join(SRC_DIR, '{}.pb.h'.format(proto_base)),
             ]
+            if ONNX_ML:
+                outputs.append(os.path.join(SRC_DIR, '{}-ml.pb.h'.format(stem)))
```

Comment:
Maybe also move the generation of "pb_py" to gen_proto.py? As it's similar to the proxy "pb.h" file.